test/Parse/statement_typo.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-typecheck-verify-swift
+
+func statementMisspelledAtTheBegginingOfTheLine() {
```

Comment:
Can you add the functions from the SR too?

Reply:
Sure. Added.